tests/Owin.Security.OpenIdConnect.Server.Tests/OAuth2AuthorizationClientCredentialsGrantTests.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Net;
+using System.Threading.Tasks;
+using Microsoft.IdentityModel.Protocols;
+using Microsoft.Owin.Testing;
+using Xunit;
+
+namespace Owin.Security.OpenIdConnect.Server.Tests {
+    public class OAuth2AuthorizationClientCredentialsGrantTests {
+        [Fact]
+        public async Task ClientCredentials_MissingCredentials_BadRequest() {
+            var server = TestServer.Create(app => { app.UseOpenIdConnectServer(new OpenIdConnectServerOptions { AllowInsecureHttp = true }); });
```

Comment:
Don't hesitate to add new lines.

You can also use the `app.UseOpenIdConnectServer(options => options.AllowInsecureHttp = true)` to make it shorter.